bin/hipify-perl

```diff
@@ -5326,6 +5326,7 @@ sub simpleSubstitutions {
     subst("channel_descriptor.h", "hip\/channel_descriptor.h", "include");
     subst("cooperative_groups.h", "hip\/hip_cooperative_groups.h", "include");
     subst("cublasLt.h", "hipblaslt.h", "include");
+    subst("math_constants.h", "hip\/hip_math_constants.h", "include");
```

Comment:
Please remove this as well, as `hipify-perl` is an autogenerated script. I'll update it in a separate change.

Reply:
Thank you for mention this. It will be generate by clang right?

Reply:
> Thank you for mention this. It will be generate by clang right?

By `hipify-clang --perl`